addons/mail/models/mail_message.py

```diff
@@ -551,6 +552,9 @@ def _get_forbidden_access(self, operation: str) -> api.Self:
         forbidden += self.browse(messages_to_check)
         return forbidden
 
+    def _get_forbidden_access_extra(forbidden, operation, documents, model, messages_to_check):
+        pass
```

Comment:
Isn't this supposed to forward existing forbidden?

But it might be more clear (and more secure) if the method only returned extra forbidden and the concat was done in the base method.